packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -2915,6 +2916,118 @@ testWidgets('SnackBarAction backgroundColor works as a Color', (WidgetTester tes
 
     expect(material.clipBehavior, Clip.antiAlias);
   });
+
+ testWidgets('Tap on button behind snack bar defined by width',
+      (WidgetTester tester) async {
```

Comment:
Consider merging this line with the one above (this is for readibility, similar to other tests descriptions in this file). Same for the second test.